start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloReloadingVFSClassLoader.java

```diff
@@ -80,6 +86,35 @@ public void run() {
           FileSystemManager vfs = AccumuloVFSClassLoader.generateVfs();
           FileObject[] files = AccumuloVFSClassLoader.resolve(vfs, uris);
```

Comment:
If the top level directory listener has not yet been removed, it will still fire create and delete events as the files change under it.  If the top level listener or ALL child listeners have been removed, these events will never fire and we never notice that something has changed.  I only opted to add in the case where the file list has been empty since it was the most common case I noticed us getting into that scenario.  We could certainly opt to call it more often.